spec/4-key-digital-functionalities.md

```diff
@@ -14,7 +14,7 @@ Following are the first core set of key functionalities of the Consent Building
 
 ### 4.1 Administrator User Functionalities
 
-The Administrator (Data Provider or Data Consumer Admin) configures the consent management system on behalf of the organisation. For simplicity, it is foreseen that one organisation involved in the data processing transaction (that is either Data Provider or Data Consumer) takes the responsibility for the configuration of the Data Policy and respective Consent Agreements(s), and so that organisation’s Administrator maintains the required configurations.
+The Administrator (Data Provider or Data Consumer Admin) configures the Consent Building Block on behalf of the organisation. For simplicity, it is foreseen that one organisation involved in the data processing transaction (that is either Data Provider or Data Consumer) takes the responsibility for the configuration of the Data Policy and respective Consent Agreements(s), and so that organisation’s Administrator maintains the required configurations.
```

Comment:
```suggestion
A Configuration Administrator (Data Provider or Data Consumer Admin) configures the Consent Building Block on behalf of the organisation. For simplicity, it is foreseen that one organisation involved in the data processing transaction (that is either Data Provider or Data Consumer) takes the responsibility for the configuration of the Data Policy and respective Consent Agreements(s), and so that organisation’s Administrator maintains the required configurations.
```